### DISCUSSION TITLE
Alien-OS Regolith

### DISCUSSION THREAD

**Discussion by Alien-Tec** [regolith-linux/regolith-desktop#910] [Show and tell]
A minimal distro based on Debian 12.1 (Kernel 6.4) with MX-Tools featuring the Regolith Desktop 3.0 with 20 different themes/looks.

![Catppuccin-Frappe](https://www.alien-os.de/userfiles/images/Alien-OS_Regolith/Catppuccin-Frappe.png)
![Dr460nf1r3](https://www.alien-os.de/userfiles/images/Alien-OS_Regolith/Dr460nf1r3.png)
![Everforest-Dark](https://www.alien-os.de/userfiles/images/Alien-OS_Regolith/Everforest-Dark.png)
![Nord](https://www.alien-os.de/userfiles/images/Alien-OS_Regolith/Nord.png)

Get it at: [https://www.alien-os.de](url)

**Comment by kgilmer** [comment]
Wow, looks cool!  The link is broken though in markdown: https://www.alien-os.de/

I'm not familiar with "MX Tools", what is it providing?

**Reply by Alien-Tec** [reply]
Thank you.
I will update the regolith-look repository and start a new pull request for the additional themes the next few days.
MX Tools is a set of applications providing a repo manager, package installer (like synaptic) with the ability to install flatpaks, too.
The best part of it is the snapshot feature which allows you to create a dvd/usb-stick of your installed system, so something like remastersys back in the days.

![MX-Tools](https://gnulinux.ch/bl-content/uploads/pages/30bd1e573e88c1b0ba2dd92257a19d2e/mx_tools_debian.png)

**Comment by ucihariskoding** [comment]
wow super cool bro !
